apps/teams/utils.py

```diff
@@ -30,24 +30,32 @@ def set_current_team(team):
         get_current_team(team)
     ```
     """
-    _context.set(team)
+    token = _context.set(team)
```

Comment:
I wonder if we should validate here that the new team is the same as the current team in the case where there is already a team set. 

I can't think of a scenario in the current code base where we would be switching teams within a single context.

Reply:
https://github.com/dimagi/open-chat-studio/pull/1436/commits/5b9be13b4753b4b6ef005bec06c96cb0d25e7c79